### DISCUSSION TITLE
editing and viewing physics in a model

### DISCUSSION THREAD

**Discussion by sreehari33** [pybamm-team/PyBaMM#2506] [Q&A]
How can I edit or view different physics in a model in PYBAMM?

**Comment by RuiheLi** [comment]
Hi~ You can easily check model options by `model.options`. For more information, you can check [this ](https://github.com/pybamm-team/PyBaMM/blob/develop/examples/notebooks/Getting%20Started/Tutorial%207%20-%20Model%20options.ipynb)source file and [this ](https://github.com/pybamm-team/PyBaMM/blob/develop/examples/notebooks/Getting%20Started/Tutorial%207%20-%20Model%20options.ipynb)notebook.
